src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -151,6 +151,24 @@ public string UniqueId
             }
         }
 
+        /// <summary>
+        /// Checks if two elements are joined
+        /// </summary>
+        /// <param name="otherElement">Element to check</param>
+        /// <returns>True if the two elements are joined, False otherwise</returns>
+        public bool IsJoined(Element otherElement)
+        {
+            if (this.InternalElement == null)
+                throw new Exception(nameof(this.InternalElement));
```

Comment:
did we still need these `null` checks ? thought the base takes care of it.
if we do, need to make them `ArgumentNullException`s